Code/Framework/AzCore/Tests/Settings/SettingsRegistryTests.cpp

```diff
@@ -1473,16 +1439,44 @@ namespace SettingsRegistryTests
         auto outcome = m_registry->MergeSettingsFile(path.Native(), AZ::SettingsRegistryInterface::Format::JsonMergePatch, {});
         EXPECT_TRUE(outcome);
 
-        AZStd::string history;
-        bool result = m_registry->Get(history, AZ_SETTINGS_REGISTRY_HISTORY_KEY "/0");
-        EXPECT_TRUE(result);
-        EXPECT_STREQ(path.c_str(), history.c_str());
-
         AZ::s64 intValue{};
         EXPECT_TRUE(m_registry->Get(intValue, "/TestObject/IntValue"));
         EXPECT_EQ(7, intValue);
     }
 
+    TEST_F(SettingsRegistryTest, MergeSettingsFile_ObjectWithImport_MergesImportedFile)
```

Comment:
If we don't have one already, it might be good to include a test that imports multiple files in the same setreg - e.g.
```
{
    "$import":"x.setreg",
    "$import":"y.setreg"
}
```